src/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp

```diff
@@ -109,18 +109,18 @@ static void regclass_graph_PreProcessSteps(py::module m) {
 
     steps.def(
         "convert_element_type",
-        [](ov::preprocess::PreProcessSteps& self, ov::element::Type type) {
+        [](ov::preprocess::PreProcessSteps& self, ov::element::Type type = {}) {
             return &self.convert_element_type(type);
         },
-        py::arg("type"),
+        py::arg("type") = ov::element::Type{},
```

Comment:
Could you provide me with print of `help(openvino.preprocess.PreProcessSteps.convert_element_type)` from the terminal?

Reply:
Help on instancemethod in module openvino.pyopenvino.preprocess:

convert_element_type(...)
    convert_element_type(self: openvino.pyopenvino.preprocess.PreProcessSteps, type: openvino.pyopenvino.Type = <Type: 'uundefined18446744073709551615'>) -> openvino.pyopenvino.preprocess.PreProcessSteps
    
    
    Converts input tensor element type to specified type.
    Input tensor must have openvino.Type data type.
    
    Parameters
    ----------
    type : openvino.runtime.Type
        Destination type. By default type will be taken from model input's element type.
    
    Returns
    ----------
    convert_element_type : PreProcessSteps
        Reference to itself to allow chaining of calls in client's code in a builder-like manner.

Reply:
As you see there is `type: openvino.pyopenvino.Type = <Type: 'uundefined18446744073709551615'>)` -- not exactly what user is expecting, this is not very informative. Moreover in every call to `convert_element_type` this temporary object will be used. I am not sure but it might be the same one in every call resulting in data races to it (no thread safety solution).

@akuporos do we have any standard way of handling such cases? If not I will try to propose some solution to this problem.

Reply:
@jiwaszki , I've made some correction and now output looks like:
```
Help on instancemethod in module openvino.pyopenvino.preprocess:

convert_element_type(...)
    convert_element_type(self: openvino.pyopenvino.preprocess.PostProcessSteps, type: openvino.pyopenvino.Type = {}) -> openvino.pyopenvino.preprocess.PostProcessSteps
    
    
    Converts tensor element type to specified type.
    Tensor must have openvino.Type data type.
    
    Parameters
    ----------
    type : Type
        Destination type. If not specified, type will be taken from model output's element type.
    
    Returns
    ----------
    convert_element_type : PostProcessSteps
        Reference to itself to allow chaining of calls in client's code in a builder-like manner.
```

Reply:
It looks a lot better now! However I would replace `{}` with something like `<Type: 'undefined'>` or `<Type: 'unspecified'>` or simple `None`. Empty brackets signalize that Python dictionary is expected which is misleading.

Reply:
help output:
```
Help on instancemethod in module openvino.pyopenvino.preprocess:

convert_element_type(...)
    convert_element_type(self: openvino.pyopenvino.preprocess.PreProcessSteps, type: openvino.pyopenvino.Type = openvino.runtime.Type.undefined) -> openvino.pyopenvino.preprocess.PreProcessSteps
    
    
    Converts input tensor element type to specified type.
    Input tensor must have openvino.Type data type.
    
    Parameters
    ----------
    type : openvino.runtime.Type
        Destination type. If not specified, type will be taken from model input's element type.
    
    Returns
    ----------
    convert_element_type : PreProcessSteps
        Reference to itself to allow chaining of calls in client's code in a builder-like manner.
```